src/components/performance-bar/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
+import { Flex } from 'rebass';
+// @ts-ignore
+import { useTheme } from 'emotion-theming';
+
+import { ITheme } from '../../theme/types';
+import * as s from './performance-bar.styles';
+
+interface Bar {
+  type: 'black' | 'gray' | 'green' | 'orange';
+  proportion: number;
+}
+
+export interface PerformanceBarProps {
+  bars: Bar[];
+}
+
+const PerformanceBar: FC<PerformanceBarProps> = ({
+  bars,
+}: PerformanceBarProps) => {
+  const theme = useTheme<ITheme>();
+  const colorMapping = {
```

Comment:
Hey, was this a situation where the theming with [variants](https://rebassjs.org/variants) did not work?
Because I have kinda the same situation on my side.